ext/native/thin3d/thin3d_gl.cpp

```diff
@@ -1037,6 +1047,8 @@ void OpenGLContext::ApplySamplers() {
 		GLenum magFilt = samp->magFilt;
 		GLenum minFilt = tex->HasMips() ? samp->mipMinFilt : samp->minFilt;
 		renderManager_.SetTextureSampler(i, wrapS, wrapT, magFilt, minFilt, 0.0f);
+		// TODO: Improve this to allow mipmaps. We don't care about those right now though for thin3d stuff.
```

Comment:
Don't we use generated mipmaps for screenshots in the UI?

-[Unknown]

Reply:
I fixed this properly.